### ISSUE TITLE
Nicer branch/tag labels in History view

### ISSUE THREAD

**Issue by AlexJitianu** [oxygenxml/oxygen-git-plugin#91]
These labels apear in the table from _History view_. They need a bit of space. Let's also try with rounded corners to see how they look.
![branches](https://user-images.githubusercontent.com/7802649/87292730-ec6a7a00-c509-11ea-8b2d-ddb0c2e3a47d.png)

These markers are added here:`com.oxygenxml.git.view.historycomponents.CommitMessageTableRenderer.addTagOrBranchLabel(List<String>, GridBagConstraints)`

**Comment by AlexJitianu** [comment]
Code review on {{com.oxygenxml.git.view.historycomponents.RoundedLineBorder}} :
1. Let's document the class, saying why we created this class. The limitations that we encountered in {{javax.swing.border.LineBorder]}
2. There are some member variables without documentation.
3. Let's put the curly brackets even if there is only one command in a "if" block. Let's use space 

` if (aaHint != null) {
        g2d.setRenderingHint(RenderingHints.KEY_ANTIALIASING, aaHint);
}`
